### PR TITLE
VR Accessibility Updates

### PR DESCRIPTION
**Pull request by Schneegans**

### CODE REVIEW

File: plugins/csp-vr-accessibility/gui/js/csp-vr-accessibility.js [hunk 1]
```diff
@@ -22,19 +22,23 @@
...
       this.picker.oninput = (e) => {
         const color = CP.HEX(e.target.value);
         this.picker.picker.set(color[0], color[1], color[2], 1);
         this.picker.style.background = CP.HEX([color[0], color[1], color[2], 1]);
-        CosmoScout.callbacks.floorGrid.setColor(color);
+
+        console.log("oninput " + e.target.value)
+        CosmoScout.callbacks.floorGrid.setColor(e.target.value);
       };
+      */
```
Comment (Schneegans) [inline plugins/csp-vr-accessibility/gui/js/csp-vr-accessibility.js:41]: Can this be re-enabled?

File: plugins/csp-vr-accessibility/README.md [hunk 2]
```diff
@@ -14,27 +14,31 @@ This plugin can be enabled with the following configuration in your `settings.js
...
+        "innerRadius": float,            // The inner radius of the vignette where the inside is 100% transparent.
+        "outerRadius": float,            // The outer radius of the vignette after which the vignette is 100% opaque.
+        "color": string,                 // The color of the vignette (as a hex-code string).
+        "fadeDuration": float,           // The time it takes for the animation to fade in the vignette in seconds.
+        "fadeDeadzone": float,           // The time of movement above the velocity threshold that is needed before the animation is played in seconds.
+        "lowerVelocityThreshold": float, // The lower threshold below which the vignette will not display (relative to Spice frame from 0..~10). 
+        "upperVelocityThreshold": float, // The upper threshold above which the vignette is always at ist minimum radius (relative to Spice frame from 0..~10).
+        "useDynamicRadius": bool,        // Toggle whether to use the dynamic vignette radius, or the fade animation with fixed radii.
+        "useVerticalOnly": bool          // Toggle whether to only draw the vignette horizontally and keep the sides unobstructed.
+      }
```
Comment (Schneegans) [inline plugins/csp-vr-accessibility/README.md:38]: As it's not enabled ber default, I would suggest to add a copy-paste example configuration to the README with some proper default values.

File: plugins/csp-vr-accessibility/gui/js/csp-vr-accessibility.js (hunk 1 above)
Comment (NXXR) [reply plugins/csp-vr-accessibility/gui/js/csp-vr-accessibility.js]: re-enabled in commit bb65637

File: plugins/csp-vr-accessibility/README.md (hunk 2 above)
Comment (NXXR) [reply plugins/csp-vr-accessibility/README.md:38]: added in commit 4f6d25c

**Review by NXXR — APPROVED** [review]

**Pull request merged**